pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -253,8 +253,84 @@ pub mod pallet {
 
 			T::Fungibles::mint_into(target_currency_id.clone(), &beneficiary, amount_to_mint)?;
 
+			Ok(())
+		}
+
+		#[pallet::call_index(2)]
+		#[pallet::weight(Weight::from_parts(10_000, 0) + T::DbWeight::get().writes(1))]
+		pub fn burn_into(
+			origin: OriginFor<T>,
+			pool_id: T::PoolId,
+			currency_idx: u32,
+			amount_to_burn: FungiblesBalanceOf<T>,
+			min_return: CollateralCurrencyBalanceOf<T>,
+			beneficiary: AccountIdLookupOf<T>,
+		) -> DispatchResult {
```

Comment:
the currencies_count parameter is still missing

Reply:
True. updated in: https://github.com/KILTprotocol/kilt-node/pull/771/files#diff-114bec8b58b7c7a182ca4abd208a4459ebd849a5e8852dddca197c043fcb1ae4R284